packages/mobile/src/transactions/CeloTransferFeedItem.test.tsx

```diff
@@ -75,6 +77,31 @@ describe('CeloTransferFeedItem', () => {
           account={''}
           comment={''}
           timestamp={1}
+          defaultImage={null}
+          defaultName={null}
+          {...getMockI18nProps()}
+        />
+      </Provider>
+    )
+    expect(tree).toMatchSnapshot()
+  })
+
+  it("renders correctly when there's a default name", () => {
```

Comment:
Nothing to do now, but wanted to point out that tests that only check the render tree via a snapshot comparison are often not very useful.
We're trying to move toward more meaningful checks via `react-native-testing-libray`.
Specifically checking for the presence of text, buttons, etc.
Example: https://github.com/valora-inc/wallet/blob/69daaa7ba298ea7daa04cd8a976c749d572e8e0f/packages/mobile/src/backup/DelayButton.test.tsx#L29